Classes/NSObject+RZImport.m

```diff
@@ -284,17 +284,37 @@ + (instancetype)rzi_objectFromDictionary:(NSDictionary *)dict withMappings:(NSDi
     NSParameterAssert(dict);
     
     id object = nil;
-    
+    BOOL didCreateNewObject = NO;
+
     if ( [self respondsToSelector:@selector( rzi_existingObjectForDict: )] ) {
         Class <RZImportable> thisClass = self;
         object = [thisClass rzi_existingObjectForDict:dict];
     }
     
     if ( object == nil ) {
+        didCreateNewObject = YES;
         object = [[self alloc] init];
     }
-    
-    [object rzi_importValuesFromDict:dict withMappings:mappings];
+
+    BOOL shouldImportValues = YES;
+
+    if ( [object respondsToSelector:@selector( rzi_shouldImportValuesFromDict:withMappings: )] ) {
```

Comment:
What's the word on following the style guide's new rules for spaces in predicates? Leave it, because this is a legacy project? Or update bit by bit?